test/nullable.jl

```diff
@@ -279,3 +279,14 @@ end
 
 # issue #11675
 @test repr(Nullable()) == "Nullable{Union{}}()"
+
+# Ensure that promotion works
+types = [
+    (DateTime, DateTime),
+    (Float64, Int64),
+]
+
+for (S, T) in types
+    @test Base.promote_type(Nullable{S}, Nullable{T}) == Nullable{Base.promote_type(S, T)}
```

Comment:
Ok, I'll update this. What operations should I be testing? Just addition and subtraction?